test/classrooms/classrooms.e2e-spec.ts

```diff
@@ -93,4 +94,39 @@ describe("ClassroomsController (e2e)", () => {
     it("returns UNAUTHORIZED(401) if user is not authenticated", () =>
       request(httpServer).get("/classrooms").expect(HttpStatus.UNAUTHORIZED));
   });
+
+  describe("GET /classrooms/:id", () => {
+    let teacherUser: User;
+    let classroom: Classroom;
+    const testUserPassword = faker.internet.password();
+
+    beforeAll(async () => {
+      teacherUser = await createSingleTeacherUserInDb(dbService, {
```

Comment:
```suggestion
      teacherUser = await createTeacherUserInDb(dbService, {
```

You can call this method multiple times if you need to create many